packages/element-app/src/components/WebKeystore/ExpansionPanelList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+// import { grey } from '@material-ui/core/colors';
```

Comment:
nit: cleanup

Reply:
leaving this for non branded case... this should really be pulled from theme, but we will wait solve that in a proper material package.

Reply:
this comment belonged on the comment above :(